mlflow/java/pom.xml

```diff
@@ -49,6 +49,26 @@
       <version>2.22.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sparkjava</groupId>
+      <artifactId>spark-core</artifactId>
+      <version>2.7.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-nop</artifactId>
+      <version>1.6.6</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-jdk14</artifactId>
+      <version>1.7.25</version>
+    </dependency>
+    <dependency>
+      <groupId>com.mashape.unirest</groupId>
```

Comment:
This package is included to provide a REST client for testing the scoring server

Reply:
Could we use org.apache.httpclient instead? We're adding that for the REST API:
https://github.com/amesar/mlflow/blob/java-client-protobuf/mlflow-java/pom.xml#L35

Reply:
Yep! Replaced unirest with the Apache http client